tools/ci/choose_workflows.py

```diff
@@ -32,6 +32,20 @@ def should_run_unit_tests(ctx: CIContext) -> bool:
     )
 
 
+def should_run_sr_unit_tests(ctx: CIContext) -> bool:
+    print('⚙️ Resolving `unit tests` phase for Session Replay:')
+    return should_run_phase(
+        ctx=ctx,
+        trigger_env=context.trigger_env.DD_RUN_SR_UNIT_TESTS,
+        build_env=context.build_env.DD_OVERRIDE_RUN_SR_UNIT_TESTS,
+        pr_keyword='[x] Run unit tests for Session Replay',
```

Comment:
I'm not including this in GH's PR template, as it won't be available until merge to `develop` and things will likely change till then.